### DISCUSSION TITLE
Cannot toggle image gen, Page freezes

### DISCUSSION THREAD

**Discussion by ComputerGuy-github** [open-webui/open-webui#6607] [Issues]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.3.35
- **Ollama (if applicable):** v0.3.14

- **Operating System:** Windows 10
- **Browser (if applicable):** Latest Opera GX

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

The settings to update

## Actual Behavior:

Page freezes and changes aren't saved

## Description

**Bug Summary:**
The page will freeze and the settings will not update

## Reproduction Details

**Steps to Reproduce:**
Go to Images page of settings and set correct settings for image gen from automatic1111 then click save

## Logs and Screenshots

**Browser Console Logs:**
[Log from Opera GX.log](https://github.com/user-attachments/files/17579930/Log.from.Opera.GX.log)

**Docker Container Logs:**
Was too long so I put in txt file
[Docker Log.txt](https://github.com/user-attachments/files/17579914/Docker.Log.txt)

**Screenshots/Screen Recordings (if applicable):**
Note: I tried many different ip's to see if that was the issue

https://github.com/user-attachments/assets/b7ceebbf-65dd-4cf4-be3e-a91f172e1506

## Additional Information

I can access the Automatic1111 UI on my device and others without issue.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by KodeMunkie** [comment]
It's the CFG value, it's not parsed properly from the automatic1111 instance. Just add ".0" to the end of the truncated value, e.g. "7.0" as a workaround. If you've already frozen the page, hit refresh and do it again but this time with the formatted float.

**Reply by ComputerGuy-github** [reply]
Thanks!
